istioctl/cmd/istioctl/deprecated_cmd.go

```diff
@@ -309,12 +312,21 @@ istioctl get virtualservice bookinfo
 			if !getByName && strings.ToLower(args[0]) == "all" {
 				typs = configClient.ConfigDescriptor()
 			} else {
-				typ, err := protoSchema(configClient, args[0])
-				if err != nil {
-					c.Println(c.UsageString())
-					return err
+				if strings.Contains(args[0], ",") {
+					var err error
+					typs, err = protoSchemaArrays(configClient, args[0])
+					if err != nil {
+						c.Println(c.UsageString())
+						return err
+					}
+				} else {
+					typ, err := protoSchema(configClient, args[0])
+					if err != nil {
+						c.Println(c.UsageString())
```

Comment:
Can we refactor `protoSchema()` so that it can adapt to multitype resource case?

Reply:
this will be more fine